easybuild/easyconfigs/v/VSim/VSim-12.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PackedBinary'
+
+name = 'VSim'
+
+version = '12.2.0'
+homepage = 'https://www.txcorp.com/vsim'
+description = "VSim is a flexible, multiplatform, multiphysics simulation software tool."
+
+toolchain = {'name': 'system', 'version': ''}
+
+sources = ['%(name)s-%(version)s-Linux64.tar.gz']
+checksums = ['d131833f76d425e94e9c6d74ce08c07cd7565950f3a58d8013b9a7d9cd179af2']
+download_instructions = 'VSim is proprietary software. Manually obtain VSim-%(version)s-Linux64.tar.gz from Tech-X (https://txcorp.com/vsim/)'
+
+sanity_check_paths = {
+    'files': ['VSimComposer.sh', 'Contents/VisIt/bin/frontendlauncher'],
+    'dirs': ['Contents']
+}
+
+modextrapaths = {
+    'PATH': 'Contents/engine/bin',
+    'LD_LIBRARY_PATH': ['Contents/VisIt/2.13.2/linux-x86_64/lib', 'Contents/engine/lib', 'Contents/lib'],
```

Comment:
I would expect most commercial software like this to take care of this via their launcher shell scripts, instead of this polluting the paths in the user environment unnecessarily.